xml-conduit/test/unit.hs

```diff
@@ -49,6 +52,7 @@ main = hspec $ do
         it "displays comments" testRenderComments
         it "conduit parser" testConduitParser
         it "can omit the XML declaration" omitXMLDeclaration
+        it "can doesn't hang on malformed entity declarations" malformedEntityDeclaration
```

Comment:
I assume you meant `it "doesn't ..."` rather than `it "can doesn't ..."`.

Reply:
Yes, fixing.